pkg/terraform/exec/plugins/vsphereprivate/resource_vsphereprivate_import_ova.go

```diff
@@ -157,16 +157,12 @@ func findImportOvaParams(client *vim25.Client, datacenter, cluster, datastore, n
 
 	// Find the network object using the provided network name
 	for _, networkMoRef := range ccrMo.Network {
```

Comment:
Do we have to do any of the lookup for the network any more? Don't we know that `networkMoRef` already?
```
networkMoRef := &ManagedObjectReference{Type: "Network", Value: network}
```

Reply:
Trying this out since DistributedVirtualPortgroup and OpaqueNetwork extend Network. Otherwise the problem is we don't know the type